pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -659,6 +659,40 @@ class Build {
         }
     }
 
+    def waitForANodeToBecomeActive(def label) {
+        def NodeHelper = context.library(identifier: 'openjdk-jenkins-helper@master').NodeHelper
+
+        if (NodeHelper.nodeIsOnline(label)) {
```

Comment:
I'm wondering if we need to change nodeIsOnline a little here.  What it's really doing is saying that there is 'a' node (perhaps out of several) that is _available_.  It's a subtle distinction but I think an important one.

Even aNodeIsOnline(label) may be more descriptive.

I could also see us enhancing NodeHelper to add a numberOfOnlineNodes(label) or some such.

Reply:
Like the function here?

https://github.com/AdoptOpenJDK/openjdk-tests/pull/1950/files